pkg/sdk/pipes_integration_test.go

```diff
@@ -0,0 +1,403 @@
+package sdk
+
+import (
+	"context"
+	"fmt"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func createCopyStatement(t *testing.T, table *Table, stage *Stage) string {
```

Comment:
it may be better to choose a more specific name like "pipeCopyStatement" because there are likely other SQL APIs with copy as an parameter name

Reply:
yes, you are right, changed.